Makefile

```diff
@@ -122,10 +122,20 @@ lint-docker:
 	@hadolint ./graders/**/Dockerfile ./workspaces/**/Dockerfile ./images/**/Dockerfile Dockerfile
 lint-shell:
 	@shellcheck -S warning $(shell find . -type f -name "*.sh" ! -path "./node_modules/*" ! -path "./.venv/*" ! -path "./testCourse/*")
+lint-sql:
+	@sqlfluff lint
```

Comment:
Is this really the best tool that exists? It's blowing my mind how slow it is, even moreso given that we've disabled so many of the rules. On my (very fast) machine, this took 37 seconds to run.

I'm looking at https://github.com/quarylabs/sqruff. Unfortunately it doesn't seem quite ready for prime time, I get some panics when running it and the list of supported rules isn't quite identical (though it's very very close). I wonder if it would be a good use of our time to try to contribute some fixes to make it work for us? With the panics, the thing runs in just a few seconds.

Reply:
I had already looked into that in https://github.com/PrairieLearn/PrairieLearn/issues/11911, but they still haven't fixed some of those outstanding issues.